python/ray/experimental/joblib/_raybackend.py

```diff
@@ -0,0 +1,54 @@
+from joblib._parallel_backends import MultiprocessingBackend
+from joblib.pool import MemmappingPool, PicklingPool
+import logging
+
+from ray.experimental.multiprocessing.pool import Pool
+import ray
+RAY_ADDRESS_ENV = "RAY_ADDRESS"
+
+logger = logging.getLogger(__name__)
+
+
+class RayBackend(MultiprocessingBackend):
+    """Ray backend uses ray, a system for scalable distributed computing.
+    more info are available here: https://ray.readthedocs.io
+    """
+
+    def configure(self,
+                  n_jobs=1,
+                  parallel=None,
+                  prefer=None,
+                  require=None,
+                  **memmappingpool_args):
+        # Make Ray Pool the father class of PicklingPool
```

Comment:
Please add a comment describing why you have to do this. Also, comments should use proper punctuation (including ending in a period)